default.json5

```diff
@@ -44,51 +44,37 @@
   ],
   packageRules: [
     {
-      matchPackagePatterns: [
-        "^io\\.confluent[.:]",
-        "^org\\.apache\\.kafka[.:]",
+      matchPackageNames: [
+        "io.confluent:*",
```

Comment:
This is different from before. Why is it not? 
`"/^io\\.confluent[.:]/"`

Reply:
This regex was never needed, we only have the following dependencies:
```
kafka-avroSerializer = { module = "io.confluent:kafka-avro-serializer", version.ref = "kafka-avroSerializer" }
kafka-clients = { module = "org.apache.kafka:kafka-clients", version.ref = "kafka-clients" }
```